tests/distributions/conftest.py

```diff
@@ -329,6 +330,11 @@ def __init__(self, rate, *, validate_args=None):
                 {'loc': [2.0, 50.0], 'scale': [4.0, 100.0],
                  'test_data': [[2.0, 50.0], [2.0, 50.0]]},
                 ]),
+    Fixture(pyro_dist=dist.SineSkewed,
+            examples=[{
+                'base_density': dist.Uniform(*tensor_wrap([-pi, -pi], [pi, pi])).to_event(1),
```

Comment:
What's the condition of base_dist? Does it need to be symmetric over a torus?...

Reply:
It should be symmetric, though it is not strictly necessary. I will also work over a circle; however, I've not checked if the distribution is proper a sphere.